hr_timesheet_project_access_restriction/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License AGPL-3
+
+===================================
+HR Timesheet No Closed Project-Task
+===================================
+
+This module adds some record rule to avoid to access on not authorized project.
```

Comment:
The module description should be improved to describe more precisely the way the security can be managed on the project.... I'm not an expert of this kind of description but something like

```
This module allow you to manage the access to projects by project:
An employee defined as 'manager' on a project can:
*  Manage the project,
*  See analytic accounts related to the project,
*  Read all timesheet line for the project .

An employee member of a project can:
* See the project,
* See analytic accounts related to the project and therefore create timesheet lines on these analytic accounts,
* See their own timesheet line related to the project.

```